src/app/games/game/game.component.ts

```diff
@@ -0,0 +1,21 @@
+import {Component, Input, OnInit} from '@angular/core';
+import {Game} from '../../store/model';
+
+@Component({
+  selector: 'app-game',
+  templateUrl: './game.component.html',
+  styleUrls: ['./game.component.scss']
+})
+export class GameComponent implements OnInit {
+
+  @Input() public game: Game;
+  private imgSrc: string;
+
+  ngOnInit(): void {
```

Comment:
Also here: ngOnInit is not needed. You can directly write `readonly imgSrc = ...`